src/order/Order.java

```diff
@@ -0,0 +1,123 @@
+package order;
+
+import address.Address;
+import food.MenuItem;
+import restaurant.Restaurant;
+import user.Driver;
+import user.User;
+
+import java.util.ArrayList;
+
+public class Order implements Comparable<Order>{
+    private User client;
+    private Restaurant restaurant;
+    private ArrayList<MenuItem> orderItems;
+    private double totalPrice;
+    private String orderStatus;
+    private Driver driver;
+
+    public Order(User client, Restaurant restaurant, ArrayList<MenuItem> orderItems, Driver driver) {
+        this.client = client;
+        this.restaurant = restaurant;
+        this.orderItems = new ArrayList<MenuItem>();

```

Comment:
here you can use diamond operator new ArrayList<> because it will infer from the left side ref declared at line 14